test/suite/test_las06.py

```diff
@@ -84,9 +84,16 @@ def test_las_reads_workload(self):
             cursor[i] = value2
         self.session.commit_transaction('commit_timestamp=' + timestamp_str(3))
 
-        # Now the latest version will get written to the data file.
+        # Write a version of the data to disk.
+        self.conn.set_timestamp('stable_timestamp=' + timestamp_str(2))
         self.session.checkpoint()
 
+        # Check the checkpoint wrote the expected values.
+        cursor2 = self.session.open_cursor(uri, None, 'checkpoint=WiredTigerCheckpoint')
+        for key, value in cursor2:
+            self.assertEqual(value, value1)
```

Comment:
Even though I checked and the code always picks the correct update (value1 = 'aaaa...') for the checkpoint, we only are not able to find all the 9999 keys here and some values end up being incorrect (value2 = 'bbbb....')